friends/src/Components/DisplayFriend.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from 'styled-components';
+// import PropTypes from 'prop-types';
+
+/***************************************************************************************************
+ ********************************************** Styles *********************************************
+ **************************************************************************************************/
+const DivWrapper = styled.div``;
+
+const DivName = styled.div`
+  display: flex;
+  justify-content: space-around;
+  font-size: 72px;
+  border: 3px solid black;
+  background-color: rgba(0, 0, 0, 0.4);
+`;
+
+const H1NotFoundMsg = styled.h1`
+  text-align: center;
+`;
+
+const ButtonEdit = styled.button`
+  margin: 10px 0;
+  border-radius: 5px;
+  outline: none;
+  background: linear-gradient(to top, #00c800, #008c00);
+  border-color: #008c00;
+  cursor: pointer;
+  width: 200px;
+  font-size: 24px;
+  font-weight: bold;
+
+  &:active {
```

Comment:
Getting really good w/ styled-components. You're styling has also gotten dramatically better since over thanksgiving break. With projects it is more important to get the functionality than the styling but it's still nice to see!